atom/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  extends: 'airbnb/base',
```

Comment:
It basically enforces the [Airbnb JS style guide](https://github.com/airbnb/javascript), which is very popular. It seemed like a good choice because it is the one that most people will be familiar with. If you want to see specifically which rules, look in these files:

- https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/legacy.js
- https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/es6.js